src/client/interface.ts

```diff
@@ -78,6 +78,12 @@ export interface WalletOptionsInterface {
 	}
 }
 
+// issuer: { requestTokens, issuerConfig }
+export interface MultiTokenInterface {
+	requestTokens: any[]
+	issuerConfig: OffChainTokenConfig
```

Comment:
@nicktaras , could you define type of requestTokens, because hard to understand why do we need "issuerConfig" as separate param.
in getTokenProofMulti() I can see code:
```const tokenCollectionConfig = userTokens[firstCollectionFound].requestTokens[0]```
could you help me to understand difference between issuerConfig and tokenConfig?

Reply:
Thanks. I found this is using the wrong interface. Ill update this.

Reply:
in MultiTokenInterface we allow multiple tokens, but single issuerConfig, is it OK? what if someone send tokens for different issuers?

Reply:
@oleggrib I've updated this to send issuer hashes to the outlet. So outlet can determine it's own config from the hash provided. This should solve this issue.